packages/@vue/cli-plugin-babel/__tests__/transpileDependencies.spec.js

```diff
@@ -70,10 +70,23 @@ test('dep from node_modules should not been transpiled', async () => {
 test('dep from node_modules should been transpiled', async () => {
   await project.write(
     'vue.config.js',
-    `module.exports = { transpileDependencies: ['external-dep'] }`
+    `module.exports = { transpileDependencies: ['external-dep', '@scope/external-dep'] }`
```

Comment:
Why do we want to implicit include scoped package with the same name? This behavior isn't documented...

Reply:
I think it was unintended… Thanks for pointing out.